ballerina/client.bal

```diff
@@ -204,6 +207,9 @@ public isolated client class Client {
         
         // Add optional headers.
         populateCreateObjectHeaders(requestHeaders, objectCreationHeaders);
+
+        //Add user defined metadata headers
```

Comment:
```suggestion
        // Add user defined metadata headers.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/pull/109/commits/6e5fb5ce2132689761aeb7a3941eccb250aecf0e